### PR TITLE
Fix/gazebo sim

### PR DESCRIPTION
**Pull request by daBaret**
Add WaA gazebo world and fix simulation parameters

### CODE REVIEW

**Comment by mantelt** [comment]
This PR now includes several things. Can't we split this up to keep a better overview?
- Adding WadA world
- adding RC based twist commands
- fixing/improving gazebo sim

**Comment by daBaret** [comment]
> This PR now includes several things. Can't we split this up to keep a better overview?
> 
>     * Adding WadA world
> 
>     * adding RC based twist commands
> 
>     * fixing/improving gazebo sim
> 
> 
> Do I miss something?
> 
> Addendum: OK, I just saw that there is #18 about the twist commands by RC. Then this is fine!

Should I create different pull requests? One per topic?

File: smb_lowlevel_controller/smb_lowlevel_controller/src/SmbHWInterface.cpp [hunk 1]
```diff
@@ -80,7 +80,11 @@ void SmbHWInterface::setDriverMode(SmbMode mode){
       bool command_smb;
       private_nh_.param<std::string>("port", port, "/dev/ttySMB");
       private_nh_.param<bool>("command_smb", command_smb, true);
-      smb_ = std::make_shared<smb_driver::SmbController>(port, private_nh_, 10, command_smb);
+
+      // RC velol$city scale
```
[the remark this answers is not in the document]
Comment (daBaret) [reply smb_lowlevel_controller/smb_lowlevel_controller/src/SmbHWInterface.cpp]: Fixed, thanks

**Comment by daBaret** [comment]
You can find it in two pull request now
- https://github.com/ETHZ-RobotX/smb_common/pull/21 for Gazebo fix
- https://github.com/ETHZ-RobotX/smb_common/pull/20 for WaA world
You can close this pull request if you want

**Comment by mantelt** [comment]
thanks a lot @daBaret! I'll close this PR then.

**Pull request closed without merging**